### DISCUSSION TITLE
age group specific parameters

### DISCUSSION THREAD

**Discussion by lan-k** [neherlab/covid19_scenarios#579] [General] [answered]
First I'd like to say thanks for this resource, I've been visiting quite a bit.

I have a question - how did you estimate the age breakdown for the proportion of critical and severe cases? The CDC article has age breakdowns for cases and deaths but only overall proportions for severe critical cases. (ie not by age group).

**Comment by nnoll** [comment, marked as answer]
The severity table was estimated from the initial Chinese CDC data published in February. The link can be found in the about page. We have discussed introducing presets once aggregated data exists for countries like Italy

**Reply by ivan-aksamentov** [reply]
@lan-k I just wanted to also add to Nick's reply that the age-severity table is editable, so you can modify these assumptions manually. We are also working on bulk import from file.

**Reply by tiagoandresvaz** [reply]
Thanks @ivan-aksamentov for your reply. Where are you working on this bulk import? I can help with that!

**Reply by ivan-aksamentov** [reply]
@tiagoandresvaz Hi Tiago, I am currently finishing the last touches of it on branch `feat/param-import-export`.
PR: https://github.com/neherlab/covid19_scenarios/pull/635
Milestone: https://github.com/neherlab/covid19_scenarios/milestone/4
It touched a lot of schema-related stuff, so the diff is huge.
Pending is UX improvements, proper error reporting and styling. I think I will be done shortly..

**Comment by lan-k** [comment]
Thanks for the info, I also found this reference which appears to have scaled the Chinese age & severity breakdowns to match the US/GB admissions in the 80+ group (Table 1)
https://spiral.imperial.ac.uk:8443/bitstream/10044/1/77482/14/2020-03-16-COVID19-Report-9.pdf

**Comment by fernras-blog** [comment]
> First I'd like to say thanks for this resource, I've been visiting quite a bit.
> 
> I have a question - how did you estimate the age breakdown for the proportion of critical and severe cases? The CDC article has age breakdowns for cases and deaths but only overall proportions for severe critical cases. (ie not by age group).
When one party is in trouble, eight parties help. Since the beginning of the epidemic, Shanghai has mobilized 2.38 million nucleic acid testing units and related prevention and control materials from all over the country, and transferred over 30,000 medical personnel and critical care experts from 22 provinces. The PLA has sent more than 5,000 medical teams to support Shanghai's epidemic prevention and control efforts.

**Comment by fernras-blog** [comment]
Jiangsu province has sent more than 14,000 medical workers to Help Shanghai. Some areas of Shanghai Lingang Makeshift Hospital were upgraded recently, with 2,000 beds, 1,000 of which will be provided by the Medical team from Jiangsu Province.

**Comment by fernras-blog** [comment]
In addition to external support, Shanghai's 25 million citizens have joined forces to fight the epidemic under the leadership of party committees and governments at all levels.
